libledger/LedgerParam.cpp

```diff
@@ -370,14 +378,12 @@ void LedgerParam::initConsensusConfig(ptree const& pt)
                                   "Please set consensus.max_trans_num to positive !"));
     }
 
-    // init maxConsensusTimeout
-    mutableConsensusParam().maxConsensusTimeout =
-        pt.get<int64_t>("consensus.max_timeout_time", 1000);
-    if (mutableConsensusParam().maxConsensusTimeout < 1000)
+    // init consensusTime
+    mutableConsensusParam().consensusTime = pt.get<int64_t>("consensus.consensus_time", 1000);

```

Comment:
consensusTimeout ？

Reply:
The `consensusTimeout` has not been replaced totally. I will fix it.